### DISCUSSION TITLE
(A little bit more) Complex layouts in quarto dashboards

### DISCUSSION THREAD

**Discussion by LuisLauM** [quarto-dev/quarto-cli#8781] [Q&A] [answered]
### Description

Hello.

I know that dashboard allows to configure the spacing through `## Row`, `## Column` and `orientation`. However, I can't find much information available to achieve a somewhat more complex layout like the one in the image:

![Untitled](https://github.com/quarto-dev/quarto-cli/assets/8452334/14146835-6ac5-4e89-ba20-80148c31bf74)

Spaces 2 through 5 are `valuebox`(es). Of course, I don't expect you to give me a ready-made solution, but I would be grateful if you could provide me with a source that I can use as a guide in addition to the one available at <https://quarto.org/docs/dashboards/layout.html>.

**Comment by mcanouil** [comment, marked as answer]
Simply nest the layout, e.g., https://github.com/mcanouil/quarto-dashboard

**Reply by LuisLauM** [reply]
Thank you so much!
